From b916a74f1674f6226fc808a8d99e4a2f00fecce7 Mon Sep 17 00:00:00 2001
From: Alex Van Damme <atv@google.com>
Date: Mon, 24 Jun 2019 13:58:26 -0700
Subject: [PATCH 03/17] Demote buggy FPS prints from g_print to GST_DEBUG.

- Port of 646ad50df63af358aa93109d1f5ded2edcd138d1 from
imx-gst-plugins-bad to imx-gst-plugins-base

Change-Id: I2894722961a5ec736a5cbe5618037fd5ae4bab14
---
 ext/gl/gstglimagesink.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ext/gl/gstglimagesink.c b/ext/gl/gstglimagesink.c
index 37a753989..07e32443b 100644
--- a/ext/gl/gstglimagesink.c
+++ b/ext/gl/gstglimagesink.c
@@ -1273,7 +1273,7 @@ gst_glimage_sink_change_state (GstElement * element, GstStateChange transition)
       glimage_sink->prev_cropmeta = NULL;
 
       if (glimage_sink->run_time > 0) {
-        g_print ("Total showed frames (%lld), playing for (%"GST_TIME_FORMAT"), fps (%.3f).\n",
+        GST_DEBUG ("Total showed frames (%lld), playing for (%"GST_TIME_FORMAT"), fps (%.3f).\n",
                 glimage_sink->frame_showed, GST_TIME_ARGS (glimage_sink->run_time),
                 (gfloat)GST_SECOND * glimage_sink->frame_showed / glimage_sink->run_time);
       }
-- 
2.26.0

